packages/client/src/links/internals/retryLink.ts

```diff
@@ -1,6 +1,6 @@
 /* istanbul ignore file -- @preserve */
 // We're not actually exporting this link
-import { AnyRouter } from '@trpc/server';
+import { AnyRouter } from '@trpc/core';
 import { observable, Unsubscribable } from '@trpc/server/observable';
```

Comment:
doesn't really matter, but should we have some "agreement" or something where we import from? the observable is an entrypoint in core pacakge as well.

what stuff is left in server that even makes it mandatory for the client to have server as a peerDep? shouldn't all imports come from core?

Reply:
yeah, most likely you're right in that all imports should be from core in our libs